code/modules/projectiles/guns/ego_gun/waw.dm

```diff
@@ -415,6 +415,19 @@
 							TEMPERANCE_ATTRIBUTE = 60,
 							JUSTICE_ATTRIBUTE = 80
 							)
+/obj/item/gun/ego_gun/bride
+	name = "bride"
```

Comment:
needs to be changed to "my own bride" or "my own wife" if the headcoder decides to use that instead.